content/news/core-monthly/2023/core-monthly-2023-01-01-2023-01-31.md

```diff
@@ -0,0 +1,617 @@
+You can [read a full announcement here]({{< relref "/news/posts/2023/github-discussions-are-now-enabled-in-the-prestashop-project" >}}).
+
+### Feature freeze for PrestaShop 8.1.0
+
+At the end of January, PrestaShop 8.1 [has entered feature freeze]({{< relref "news/releases/prestashop-8-1-feature-freeze" >}}), meaning no new features will be added, and the focus is now on polishing existing features before release. A public beta is expected to be released in a couple of weeks, and the community is encouraged to participate in testing and provide feedback. The most notable features in PrestaShop 8.1 include improved product pages, image generation in additional formats, better control over discontinued products, and more. I encourage you to [subscribe to the GitHub issue](https://github.com/PrestaShop/PrestaShop/issues/30719), so you can stay updated on the release of 8.1.
+
+That also means that work on PrestaShop 9.0 has begun! Contributions introducing backward-incompatible changes can now be merged into the `develop` branch for the next major version. If you have any ideas and want to contribute [we can discuss them](https://github.com/PrestaShop/PrestaShop/discussions)! Make sure to follow project communication channels for more updates on PrestaShop 9.0 soon.
+
+### Module releases
+
+As you can see later in the article, we released many new versions of the modules in January. It wouldn't be possible without a huge work from [Daniel Hlavacek (@Hlavtox)](https://github.com/Hlavtox) and our amazing Quality Assurance team! You can see a release of a `ps_viewedproduct` module, with a [fix for a popular bug with PrestaShop cookie](https://github.com/PrestaShop/ps_viewedproduct/pull/25) that was reported many times. There's also a [new version](https://github.com/PrestaShop/ps_emailsubscription/releases/tag/v2.7.1) of `ps_emailsubscription` with some [performance optimalizations](https://github.com/PrestaShop/ps_emailsubscription/pull/89) and an improvement, so that developres can [perform additional actions after customer's subscription during registration](https://github.com/PrestaShop/ps_emailsubscription/pull/84/files).
+
+### Community events
+
+In January, we held [the first Live Update of the year]({{< relref "/news/public-oss-demo/2023/live-update-1-2023" >}}). We presented [a new feature that is coming to PrestaShop 8.1](https://youtu.be/YDS51o48W1I?t=1805) and which will allow you to generate real WebP and Avif images. It was also a bit sad event because it was the last Live Update with [Valentin (@NeOMaking)](https://github.com/NeOMaking) as a PrestaShop SA employee. He presented [Hummingbird](https://github.com/PrestaShop/hummingbird) theme, which will, hopefully, be the default theme for PrestaShop 9.0. Valentin is now enjoying his new adventure in the industry, but he confirmed he would continue supporting the project with his knowledge.
```

Comment:
```suggestion
In January, we held [the first Live Update of the year]({{< relref "/news/public-oss-demo/2023/live-update-1-2023" >}}). We presented [a new feature that is coming to PrestaShop 8.1](https://youtu.be/YDS51o48W1I?t=1805) and which will allow you to generate real WebP and Avif images. It was also a bit of a sad event because it was the last Live Update with [Valentin (@NeOMaking)](https://github.com/NeOMaking) as a PrestaShop SA employee. He presented the [Hummingbird](https://github.com/PrestaShop/hummingbird) theme, which will, hopefully, be the default theme for PrestaShop 9.0. Valentin is now enjoying his new adventure in the industry, but he confirmed he would continue supporting the project with his knowledge.
```